### DISCUSSION TITLE
Best Blazor training courses and books?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [MarimerLLC/csla#2385] [General]
Hi everyone

Apart from Rocky's book, _Using CSLA 5: Blazor and WebAssembly_, can you recommend any good courses or books?

**Comment by danielmartind** [comment]
I recommend Blazor in Action by Chris Sainty, currently a MEAP at manning.com

**Reply by rockfordlhotka** [reply]
Chris is really knowledgeable! He also has the [blazord project](https://github.com/Blazored).

[Michael Washington](https://mvp.microsoft.com/en-us/PublicProfile/38654) also has some books and websites worth checking out.

**Reply by danielmartind** [reply]
[Blazor Train](https://blazortrain.com/) is worth checking out too.
